doc/grass_database.html

```diff
@@ -52,28 +53,32 @@ <h3>GRASS GIS Database</h3>
 inside in a specific format and is accessible by specific tools.
 GRASS GIS Databases is in GRASS GIS often called GISDBASE or DATABASE.
 
-<h3>GRASS Locations</h3>
+<h3>GRASS Projects</h3>
 
-Location is a directory which contains GRASS Mapsets which are its subdirectories.
-All data in one Location have the same projection (coordinate system, datum).
-Each Location must contain Mapset called PERMANENT.
-Typically, a Location contains all data related to one project
+Project is a directory which contains GRASS Mapsets which are its subdirectories.
+All data in one Project have the same coordinate reference system (projection, datum).
+Each project must contain Mapset called PERMANENT.
+Typically, a project contains all data related to one real-world project
 or a geographic area (geographic location or region).
```

Comment:
This definition of project with project sounds odd to me, but I do not have a better suggestion right now

Reply:
I'm aiming for a minimal changes needed for the rename in this PR, but having this PR in, perhaps more people can start improving the documentation.